### DISCUSSION TITLE
Update library via Web Gui

### DISCUSSION THREAD

**Discussion by fmnamado** [xthursdayx/yacreaderlibrary-server-docker#46] [Ideas]
Hello,

First of all thank you for your docker.
I am making this post to serve as a question and as a suggestion.

Is it possible to make and update library button, aswell as some king of display the progress of the update, in the WebGUI?
I am referring as something similar as Plex implemented this. You can just click update, and you can see in the upper right section the progress of library being updated. This serve as a debug (to see if it is updating or not).

It would be way simpler to me to update.
Something I transfer new files of the network using my phone.
It is easier to access the webserver and click update, than having to connect via ssh in the phone, copy and pasting a big command... 

I think I made myself understood in the idea, right?

Thank you once again!

**Comment by xthursdayx** [comment]
Hello, this is an excellent suggestion, but it's one that should be made on the YACReader repo, since the YACReader Library Server WebUI is programmed into the base software, not this docker image.

**Comment by fmnamado** [comment]
Hello,
Thank you for your reply.

I just posted it here https://github.com/YACReader/yacreader/issues/328

Hope it is on the right place 😄
